### ISSUE TITLE
bower error installing on Linux Mint

### ISSUE THREAD

**Issue by mr-david** [wlaurance/slack-for-linux#57]
```
dcolon@athena ~ $ sudo npm install slack-for-linux -g
<snip>
+ ../node_modules/.bin/bower install
bower ESUDO         Cannot be run with sudo

Additional error details:
Since bower is a user command, there is no need to execute it with superuser permissions.
If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.

http://www.joyent.com/blog/installing-node-and-npm
https://gist.github.com/isaacs/579814

You can however run a command with sudo using --allow-root option
```

I've read that I need to add the --allow-root option to a json file but it's not clear which one. I've tried adding it to ~/.npm/slack-for-linux/1.4.1/package/install.sh (obviously not a json file) but I get the same error.

**Comment by twolfson** [comment]
This is being patched in #54 by removing the additional install steps altogether. Please try out the PR's preview via:

```bash
npm install -g twolfson/slack-for-linux#6a99eb6050e3e97b79a44cc0df1c56c04c2ca4dc
```

**Comment by mr-david** [comment]
Thanks for the quick response. I'm brand new to Node and would like to know which file to add the "--allow-root" option. 

I will give your preview release a shot as well. Thanks

**Comment by twolfson** [comment]
We tried out going down that path and encountered even more errors. I believe we tried adding it to `./install.sh`. That is why we went for the single install approach in #54.

**Comment by mr-david** [comment]
@twolfson, thank you the preview is working.

**Comment by wlaurance** [comment]
@mr-david Hi, #54 has been released and is in slack-for-linux@latest (1.4.3)

Since the preview worked for you, I'll close this issue.
